### PR TITLE
Use gobwas/ws instead of gorilla/webtorrent

### PR DESCRIPTION
**Pull request by gitpubber**
https://github.com/anacrolix/torrent/pull/503

But,this one uses net.Conn instead of *net.Conn

### CODE REVIEW

**Comment by gitpubber** [comment]
@anacrolix ,please review this

**Comment by gitpubber** [comment]
also, please release next version after merging this(after reviewing) .

thanks

File: webtorrent/tracker_client.go [hunk 1]
```diff
@@ -130,9 +133,7 @@ func (tc *TrackerClient) Run() error {
 func (tc *TrackerClient) Close() error {
 	tc.mu.Lock()
 	tc.closed = true
-	if tc.wsConn != nil {
-		tc.wsConn.Close()
```
Comment (anacrolix) [inline webtorrent/tracker_client.go:134]: Probably still need this.

**Review by anacrolix — CHANGES_REQUESTED** [review]
Looks promising. Have you tested it with Webtorrent peers?

**Comment by gitpubber** [comment]
@anacrolix https://github.com/anacrolix/torrent/pull/505

**Comment by gitpubber** [comment]
> Looks promising. Have you tested it with Webtorrent peers?

yes. but, you also once test it if you want

File: webtorrent/tracker_client.go (hunk 1 above)
Comment (gitpubber) [reply webtorrent/tracker_client.go:134]: https://github.com/anacrolix/torrent/pull/505

**Pull request closed without merging**